Detector/DetFCChhHCalTile/src/HCalBarrel_geo.cpp

```diff
@@ -105,10 +105,10 @@ static DD4hep::Geometry::Ref_t createHCal (
   double dy0 = moduleDimensions.dz();
   double dz0 = moduleDimensions.dr() / 2.;
 
-  double drWedge = (dimensions.rmax() - dimensions.rmin()) * 0.5;
+  double drWedge = cos(dphi / 2.) * (dimensions.rmax() - dimensions.rmin()) * 0.5;
```

Comment:
Hi Jana,
right! Thanks for spotting this, I misread the plate dimensions. Ok, I will try to implement your suggestion so the plates are contained by the envelope.
Cheers,
Valentin